File: python/src/Behavioral/Command/DvdCommandNameStarsOff.py
#//DvdCommandNameStarsOff - two of two Concrete Commands

from CommandAbstract import CommandAbstract

class DvdCommandNameStarsOff(CommandAbstract) :

	def __init__(self,dvdName) :
		self.dvdName = dvdName

	def execute(self) : 
		self.dvdName.setNameStarsOff()

File: ruby/src/Behavioral/Command/DvdCommandNameStarsOff.rb
#//DvdCommandNameStarsOff - two of two Concrete Commands

require "CommandAbstract"

class DvdCommandNameStarsOff < CommandAbstract

	def initialize(dvdName) 		@dvdName = dvdName

	def execute(self) 		@dvdName.setNameStarsOff()
